scripts/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
would this be better served in a `gemspec`?
almost feels like the script directory is becoming a gem